tests/load_cache.test/runit

```diff
@@ -0,0 +1,249 @@
+    typeset f=$1
+    write_prompt $func "$f failed: $2"
+    cleanup
+    exit -1
+}
+
+function bulk_insert_records
+{
+    [[ $debug == "1" ]] && set -x
+    typeset func="bulk_insert_records"
+    write_prompt $func "Running $func"
+    typeset table=${1:-t1}
+    typeset cnt=${2:-100000}
+    typeset chunk=${3:-10000}
+    j=0
+    while [[ $cnt -gt 0 ]]; do
+        if [[ $cnt -lt $chunk ]]; then
+            amt=$cnt
+        else
+            amt=$chunk
+        fi
+        $CDB2SQL_EXE -tabs $CDB2_OPTIONS $DBNAME default "insert into $table select * from generate_series(1, $amt)" >/dev/null
+        let cnt=$(( cnt - amt ))
+    done
+}
+
+function flush_cluster
+{
+    [[ $debug == "1" ]] && set -x
+    typeset func="flush_cluster"
+    write_prompt $func "Running $func"
+
+    if [[ -z "$CLUSTER" ]]; then
+        $CDB2SQL_EXE $CDB2_OPTIONS --tabs $DBNAME default "exec procedure sys.cmd.send(\"flush\")"
+        $CDB2SQL_EXE $CDB2_OPTIONS --tabs $DBNAME default "exec procedure sys.cmd.send(\"flush_pagelist\")"
+    else
+        for n in $CLUSTER ; do 
+            $CDB2SQL_EXE $CDB2_OPTIONS --tabs $DBNAME --host $n "exec procedure sys.cmd.send(\"flush\")"
+            $CDB2SQL_EXE $CDB2_OPTIONS --tabs $DBNAME --host $n "exec procedure sys.cmd.send(\"flush_pagelist\")"
+        done
+    fi
+}
+
+function setup
+{
+    [[ $debug == "1" ]] && set -x
+    typeset func="setup"
+    write_prompt $func "Running $func"
+    drop_table
+    create_table
+}
+
+function load_cache
+{
+    [[ $debug == "1" ]] && set -x
+    typeset func="load_cache"
+    write_prompt $func "Running $func"
+    typeset file=${1:-cache.txt}
+    if [[ -z "$CLUSTER" ]]; then
+        $CDB2SQL_EXE $CDB2_OPTIONS --tabs $DBNAME default "exec procedure sys.cmd.send(\"load_cache $file\")"
+    else
+        for n in $CLUSTER ; do 
+            $CDB2SQL_EXE $CDB2_OPTIONS --tabs $DBNAME --host $n "exec procedure sys.cmd.send(\"load_cache $file\")"
+        done
+    fi
+}
+
+function dump_cache
+{
+    [[ $debug == "1" ]] && set -x
+    typeset func="dump_cache"
+    write_prompt $func "Running $func"
+    typeset file=${1:-cache.txt}
+    typeset dbname=${2:-$DBNAME}
+    if [[ -z "$CLUSTER" ]]; then
+        $CDB2SQL_EXE $CDB2_OPTIONS --tabs $dbname default "exec procedure sys.cmd.send(\"dump_cache $file\")"
+    else
+        export first=1
+        for n in $CLUSTER ; do 
+            $CDB2SQL_EXE $CDB2_OPTIONS --tabs $dbname --host $n "exec procedure sys.cmd.send(\"dump_cache $file\")"
+            scp $SSH_OPT ${n}:${file} ${file}
+            cp ${file} ${file}.${n}
+        done
+    fi
+}
+
+function sort_and_diff_files
+{
+    [[ $debug == "1" ]] && set -x
+    typeset func="diff_files"
+    write_prompt $func "Running $func"
+    typeset file1=$1
+    typeset file2=$2
+    typeset linecnt
+    typeset faildiff
+    typeset cnt
+    linecnt=$(wc -l ${file1} | awk '{print $1}')
+    [[ -z $linecnt ]] && failexit "Cant read cache file"
```

Comment:
Several problems here .. linecnt should be calculated against the sorted file .. cat file2 should be redirected to file2.sort